roles/openshift_health_checker/library/docker_container.py

```diff
@@ -0,0 +1,2032 @@
+#!/usr/bin/python
+# pylint: skip-file
+#
+# This file is a copy of https://github.com/ansible/ansible/commit/20bf02f6b96356ab5fe68578a3af9462b4ca42a5
+# It has been temporarily vendored here due to issue https://github.com/ansible/ansible/issues/22323
+# We can remove this file once openshift-ansible requires ansible > 2.2.1.0.
```

Comment:
- Please add one or two blank comment lines here to keep it separate from the copyright notice. Maybe some ASCII art to make the distinction clear, use your own judgement:

  ```python
  # TODO: remove ....
  #
  # -------------------------------------------------------------------
  #
  # Copyright 2016 ...
  ```

- I suggest marking the TODO with a "TODO" comment, those are easier to find by grepping the code base than generic comments. Hopefully that would increase the likelihood we'll remove this file in the future.